packages/mui-material/src/Backdrop/Backdrop.js

```diff
@@ -49,8 +49,6 @@ const Backdrop = React.forwardRef(function Backdrop(inProps, ref) {
     children,
     className,
     component = 'div',
-    components = {},
-    componentsProps = {},
```

Comment:
@DiegoAndai, so thanks to your wisdom all the tests are passing now! 

The only thing to mention was that now components and componentsProps is unused...  Which makes sense, its the aim of the MR. I just wanted to check before I go removing stuff.